packages/create-cloudflare/templates/hello-world/js/test/index.spec.js

```diff
@@ -0,0 +1,18 @@
+import { env, createExecutionContext, waitOnExecutionContext, SELF } from "cloudflare:test";
+import { describe, it, expect } from "vitest";
+import worker from "../src";
+
+describe("Hello World worker", () => {
+  it("displays Hello World! (unit style)", async () => {
```

Comment:
Strictly it responds with, rather than displays?

```suggestion
  it("responds with Hello World! (unit style)", async () => {
```

And similarly for the other test?

Reply:
Updated 👍